lib/i18n/tasks/translators/deepl_translator.rb

```diff
@@ -17,7 +17,12 @@ def initialize(*)
     protected
 
     def translate_values(list, from:, to:, **options)
-      DeepL.translate(list, to_deepl_compatible_locale(from), to_deepl_compatible_locale(to), options).map(&:text)
+      deepl = DeepL.translate(list, to_deepl_compatible_locale(from), to_deepl_compatible_locale(to), options)
+      if deepl.respond_to? 'map'
```

Comment:
Does it respond with a string when there is only one element in the list?
If so, it'd be cleaner to check the list instead of checking the result: consider what happens if a later version of Ruby adds a `map` method to `String`

Reply:
What about using Ruby goodness?

```
result = DeepL.translate(list, to_deepl_compatible_locale(from), to_deepl_compatible_locale(to), options)
[*result].map(&:text)
```

The splat operator would leave it all to Ruby ;-)